src/view/com/posts/FeedSlice.tsx

```diff
@@ -27,6 +33,7 @@ let FeedSlice = ({slice}: {slice: FeedPostSlice}): React.ReactNode => {
           moderation={slice.items[0].moderation}
           isThreadParent={isThreadParentAt(slice.items, 0)}
           isThreadChild={isThreadChildAt(slice.items, 0)}
+          hideTopBorder={hideTopBorder}
```

Comment:
In feeds, we use `FeedSlice` instead of `FeedItem` directly. So, we need to tell the first item in the feed slice to not display the border _if_ it is also the first item in `renderItem`.